src/styles/card.scss

```diff
@@ -0,0 +1,124 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: $card-width;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: calc($card-width - 40px);
+    height: 134px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: $font-size-base;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 $small-margin;
+    color: #060b35;
+  }
+
+  &__code {
+    width: 100%;
+    display: flex;
+    justify-content: flex-start;
+    font-weight: 400;
+    color: $secondary-color;
+    margin-bottom: $medium-margin;
```

Comment:
Similar to the `$small-margin`, ensure that `$medium-margin` is defined appropriately to match the design requirements for medium-sized margins.